omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ServerLogResource1_8.java

```diff
@@ -0,0 +1,88 @@
+import io.swagger.models.properties.MapProperty;
+import org.openmrs.module.webservices.helper.ServerLogActionWrapper;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.annotation.Resource;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.Listable;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+
+/**
+ * {@link Resource} for ServerLogController, supporting standard CRUD operations
+ */
+@Resource(name = RestConstants.VERSION_1 + "/serverlog", supportedClass = ServerLogActionWrapper.class, supportedOpenmrsVersions = {
+        "1.8.*", "1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*" })
+public class ServerLogResource1_8 extends BaseDelegatingResource<ServerLogActionWrapper> implements Listable {
+	
+	private ServerLogActionWrapper serverLogActionWrapper = new ServerLogActionWrapper();
+	
+	public void setServerLogActionWrapper(ServerLogActionWrapper serverLogActionWrapper) {
+		this.serverLogActionWrapper = serverLogActionWrapper;
+	}
+	
+	@Override
+	public SimpleObject getAll(RequestContext context) throws ResponseException {
+		SimpleObject rest = new SimpleObject();
+		rest.put("serverLog", serverLogActionWrapper.getServerLogs());
+		return rest;
+	}
+	
+	@Override
+	public Model getGETModel(Representation rep) {
+		return ((ModelImpl) super.getGETModel(rep))
+		        .property("serverLog", new MapProperty());
+	}
+	
+	@Override
+	public ServerLogActionWrapper getByUniqueId(String uniqueId) {
+		throw new UnsupportedOperationException("Serverlog doesn't support to this action");
+	}
+	
+	@Override
+	protected void delete(ServerLogActionWrapper delegate, String reason, RequestContext context) throws ResponseException {
+		throw new UnsupportedOperationException("Serverlog doesn't support to this action");
+	}
+	
+	@Override
+	public ServerLogActionWrapper newDelegate() {
+		return new ServerLogActionWrapper();
+	}
+	
+	@Override
+	public ServerLogActionWrapper save(ServerLogActionWrapper delegate) {
+		throw new UnsupportedOperationException("Serverlog doesn't support to this action");
+	}
+	
+	@Override
+	public void purge(ServerLogActionWrapper delegate, RequestContext context) throws ResponseException {
+		throw new UnsupportedOperationException("Serverlog doesn't support to this action");
+	}
+	
+	@Override
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		description.addProperty("serverLog", "serverLog");
```

Comment:
The properties should be the individual fields on the ServerLogWrapper class

Reply:
I have added this property to the `ServerLogActionWrapper` class here - https://github.com/openmrs/openmrs-module-webservices.rest/pull/329/files#diff-410f1f6f7c035020e68aa6abe419b84bR28

Do we need another `ServerLogWrapper` Class also?